label_maker/utils.py

```diff
@@ -14,3 +22,38 @@ def class_match(ml_type, label, i):
     elif ml_type == 'segmentation':
         return np.count_nonzero(label == i)
     return None
+
+def download_tile_tms(tile, imagery, dest_folder):
+    """Download a satellite image tile from a tms endpoint"""
+    o = urlparse(imagery)
+    _, image_format = op.splitext(o.path)
+    r = requests.get(url(tile.split('-'), imagery))
+    tile_img = op.join(dest_folder, 'tiles', '{}{}'.format(tile, image_format))
+    open(tile_img, 'wb').write(r.content)
+
+def get_tile_tif(tile, imagery, dest_folder):
```

Comment:
Add a few more comments within this function

Reply:
Sorry, I meant within the function itself :p

Just 2-3 lines to say what the `proj_to` functions are doing and how the `window`is being used.